app/src/main/java/org/ole/planet/myplanet/ui/exam/TakeExamFragment.java

```diff
@@ -78,15 +78,16 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
         initExam();
         questions = mRealm.where(RealmExamQuestion.class).equalTo("examId", exam.getId()).findAll();
-        tvQuestionCount.setText("Question : " + currentIndex +" /" + questions.size());
+        boolean hasQuestions = questions.size() > 0;
+        tvQuestionCount.setText("Question : " + (hasQuestions ? "1" : currentIndex) +" /" + questions.size());
```

Comment:
Just wondering whenever there is question wouldn't this statement always shows 1/questionsize? If so why are we putting currentIndex?

Reply:
@lmmrssa nope, if it has no question it will be `0/0` before it seems like 1/0 even if there is no question.When the user answers the question the the value of currentIndex will be increased and the text is changed again in line no 117 of `TakeExamFragment.java`

` @Override
    public void startExam(RealmExamQuestion question) {
        tvQuestionCount.setText("Question : " + (currentIndex + 1) + "/" + questions.size());`

`currentIndex was initially set to value 0 so at first when fragment loads it will show as 0/0 0r 1/question_size`

Reply:
When we are not allowing user to take exam when there is no question why do we need to handle case of no question?

Reply:
@lmmrssa i am reverting the changes in `TakeExamFragment`